acceptance/main_test.go

```diff
@@ -48,24 +48,24 @@ import (
 	"github.com/cockroachdb/cockroach/util/stop"
 )
 
-var duration = flag.Duration("d", 5*time.Second, "duration to run the test")
-var numLocal = flag.Int("num-local", 0, "start a local cluster of the given size")
-var numRemote = flag.Int("num-remote", 0, "start a remote cluster of the given size")
-var numStores = flag.Int("num-stores", 1, "number of stores to use for each node")
-var cwd = flag.String("cwd", "../cloud/aws", "directory to run terraform from")
-var stall = flag.Duration("stall", 2*time.Minute, "duration after which if no forward progress is made, consider the test stalled")
-var keyName = flag.String("key-name", "", "name of key for remote cluster")
-var logDir = flag.String("l", "", "the directory to store log files, relative to the test source")
+var flagDuration = flag.Duration("d", 5*time.Second, "duration to run the test")
+var flagNodes = flag.Int("nodes", 1, "number of nodes")
```

Comment:
Do the tests work when run with one node? I think 3 would be a more useful default. 

Remove "remote" from the help text.